lib/src/widgets/switch/ds_switch_base.widget.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSSwitchBase extends StatelessWidget {
```

Comment:
Use ```abstract``` keyword so the ```build``` method doesn't need any implementation.

Reference: https://dart.dev/samples#interfaces-and-abstract-classes